ivy/functional/frontends/tensorflow/metrics.py

```diff
@@ -0,0 +1,15 @@
+import ivy
+
+
+def sparse_categorical_crossentropy(
+    y_true,
+    y_pred,
+    from_logits=False,
+    axis=-1
+):
+    if from_logits:
+        y_pred = ivy.softmax(y_pred)
+    return ivy.sparse_cross_entropy(y_true, y_pred, axis=axis, epsilon=1e-7)
```

Comment:
As epsilon's default value is 1e-7 in ivy.sparse_cross_entropy, so you may just skip this keyword argument.